packages/strapi/lib/commands/build.js

```diff
@@ -26,10 +26,50 @@ module.exports = async ({ clean, optimization }) => {
     process.exit(1);
   }
 
-  const serverConfig = await loadConfigFile(envConfigDir, 'server.+(js|json)');
+  const conf = await loadConfigFile(envConfigDir, 'server.+(js|json)');
 
-  const adminPath = _.get(serverConfig, 'admin.path', '/admin');
-  const adminBackend = _.get(serverConfig, 'admin.build.backend', '/');
+  // Defines serverUrl value
+  let serverUrl = _.get(conf, 'url', '');
+  serverUrl = _.trim(serverUrl, '/ ');
+  if (typeof serverUrl !== 'string') {
```

Comment:
Why do you need this check? can't you use the statement below directly?

Reply:
Because if it doesn't start with 'http' it would be concatenated line 46. Do you have a suggestion?

Reply:
Maybe you can simply use the statement line 39?

Reply:
You want me to make this `if` a `else if`?